app/Config/Mimes.php

```diff
@@ -516,11 +516,7 @@ public static function guessExtensionFromType(string $type, ?string $proposed_ex
 
 		foreach (static::$mimes as $ext => $types)
 		{
-			if (is_string($types) && $types === $type)
-			{
-				return $ext;
-			}
-			else if (is_array($types) && in_array($type, $types))
+			if ((is_string($types) && $types === $type) || (is_array($types) && in_array($type, $types)))
```

Comment:
Please format code. Code looks ugly!

Reply:
Ugly? The code is optimisation of different if conditions. It's not meant to showcase the multiline beautiful reordering. Optimised code is real beauty